database/runtime/handler.py

```diff
@@ -183,6 +183,63 @@ def create_collection_summaries_functions(cursor) -> None:
     """
     cursor.execute(sql.SQL(periodic_datetime_summary_sql))
 
+    search_collections_sql = """
```

Comment:
nit: can we put the search collections definition in its own `create_search_collections_functions()` instead of inside of the `create_collection_summaries_functions()` just for future readability? 

Or if we want to fit all the collection-sh function definitions into one long function maybe shift the summary function to the end (instead between the datetime functions) and update the docstrings?
---
Also: have you tried this out in the dev stack? I'm tempted to just add the function manually to try it out--can always drop it after testing and increment the DB SCHEMA version to make sure the pgstac bootstrapper deployment runs the new definition

Reply:
Yeah, I've tested the function out and I *believe* it does what is expected. Should work on dev or even locally if you have an instance

Reply:
Good point, I'll run locally!